src/client.ts

```diff
@@ -0,0 +1,132 @@
+import * as request from "request-promise-native";
+
+interface assoc {
+    [key: string]: any;
+}
+
+export class Client {
+
+    endpoint: string = 'https://appwrite.io/v1';
```

Comment:
1. `string` will be inferred automatically, you don't need to mention it.
2. also, consider prefixing it with `private readonly` to be on the safe side, if it fits your intentions.